lib/internal/Magento/Framework/File/Test/Unit/UploaderTest.php

```diff
@@ -60,10 +54,6 @@ public function getCorrectFileNameProvider()
             [
                 'a.' . str_repeat('b', 88),
                 'a.' . str_repeat('b', 88)
-            ],
-            [
-                'a.' . str_repeat('b', 89),
-                true
```

Comment:
could you explain why you removed it? Maybe it's better to cover the case when it will start failing?

Reply:
Because I removed from lib/internal/Magento/Framework/File/Uploader.php:403, in this case the method isn't going to throw no exception

`
        // account for excessively long filenames that cannot be stored completely in database
        if (strlen($fileInfo['basename']) > 90) {
            throw new \InvalidArgumentException('Filename is too long; must be 90 characters or less');
        }
`
So then we don't need on lib/internal/Magento/Framework/File/Uploader.php:24 tests exception cases.